Dockerfile

```diff
@@ -50,6 +52,6 @@ ADD unlock_emulator.sh /bin/unlock_emulator
 RUN chmod +x /bin/unlock_emulator
 
 #Label
-MAINTAINER Nilton Vasques <nilton.vasques@gmail.com>
-LABEL Version="0.1.5" \
+MAINTAINER Flavio Kreis <flaviokreis@gmail.com>
```

Comment:
You need update with your information

Reply:
yes! but I can't here on this PR. I will change on master.